src/components/forms/forms.js

```diff
@@ -14,24 +15,131 @@ import {
 import './forms.css';
 
 export const FormFieldTypes = {
-  MULTITEXT: { defaultValue: [], component: formInputMultiText },
-  SLIDER: { defaultValue: false, component: formInputSlider },
-  RADIO: { defaultValue: null, component: formInputRadioGroup },
-  TEXT: { defaultValue: '', component: formInputGeneric },
-  NUMBER: { defaultValue: '', component: formInputGeneric },
+  MULTITEXT: {
+    defaultValue: [],
+    component: formInputMultiText,
+    requiredProps: [],
+    possibleProps: [
+      'title',
```

Comment:
should possible props always contain these common props? The ones that are possible for all form field types?